.circleci/config.yml

```diff
@@ -7,24 +7,11 @@ orbs:
   rust: circleci/rust@1.6.1
 
 commands:
-  notify-failures-on-main:
+  notify-failures-to-slack:
```

Comment:
Nit: shall we still have "on main" in the name, since that is a hardcoded branch pattern?

Reply:
restored.